tt_metal/core_descriptors/blackhole_140_arch_eth_dispatch.yaml

```diff
@@ -6,30 +6,31 @@
 #     core descriptor config
 
 blackhole:
-  1:
-    compute_with_storage_grid_range: # Logical only start and end [x, y]
-      start: [0, 0]
-      end: [13, 9]
+  row:
```

Comment:
we should assert that col dispatch is not allowed when doing eth dispatch on BH because there is only one row of eth cores